sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Constants.cs

```diff
@@ -187,7 +187,7 @@ internal class Constants
     // Default constants
     internal static readonly string s_default_os = ServiceOs.Linux;
     internal static readonly string s_default_expose_network = "<loopback>";
-    internal static readonly string s_pLAYWRIGHT_SERVICE_DEBUG = "PLAYWRIGHT_SERVICE_DEBUG";
+    internal static readonly string s_pLAYWRIGHT_SERVICE_DEBUG = "Logging__LogLevel__MicrosoftPlaywrightTesting";
```

Comment:
Logging.LogLevel.MicrosoftPlaywrightTesting is this consistent with js package, ? if not then should we ? Does the OSS follow level pattern in thier DEBUG variable ? I wonder if we could follow similar pattern ?

Reply:
> is this consistent with js package

No, this is Azure SDK .NET standard

Reply:
https://learn.microsoft.com/en-us/dotnet/azure/sdk/logging